gotestcover.go

```diff
@@ -45,13 +48,32 @@ func run() error {
 	if err != nil {
 		return err
 	}
+
+	coverage := 0.0
+	coveredPackages := 0
+	coverageRe := regexp.MustCompile(`coverage: (\d+\.\d+)%`)
+
 	cov := runAllPackageTests(ps, func(out string) {
 		fmt.Print(out)
+
+		if flagCoverageSummary {
+			if m := coverageRe.FindStringSubmatch(out); m != nil {
+				c, _ := strconv.ParseFloat(m[1], 32)
+				coverage += c
+				coveredPackages++
+			}
+		}
 	})
+
+	if flagCoverageSummary {
+		fmt.Printf("Coverage Summary: %f%%\n", coverage/float64(coveredPackages))
```

Comment:
Well, give me the means to count covered/not covered lines and I will do a better percentage, but I will not start changing Go upstream again for such a small thing.